crates/viewer/re_selection_panel/src/item_title.rs

```diff
@@ -76,12 +77,14 @@ impl ItemTitle {
         let id_str = format!("{} ID: {}", store_id.kind, store_id);
 
         let title = if let Some(entity_db) = ctx.store_context.bundle.get(store_id) {
-            if let Some(info) = entity_db.store_info() {
-                let time = re_log_types::Timestamp::from(info.started)
+            if let (Some(application_id), Some(started)) =
+                (entity_db.app_id(), entity_db.property::<Timestamp>())
+            {
+                let time = re_log_types::Timestamp::from(started.0)
                     .to_jiff_zoned(ctx.app_options().timestamp_format)
                     .strftime("%H:%M:%S")
                     .to_string();
-                format!("{} - {}", info.application_id, time)
+                format!("{application_id} - {time}")
             } else {
                 id_str.clone()
```

Comment:
if application id is available but no `Timestamp`, shouldn't we at least use application id in the title?